fv3core/fv3core/stencils/dyn_core.py

```diff
@@ -29,16 +29,10 @@
 from fv3core.stencils.pk3_halo import PK3Halo
 from fv3core.stencils.riem_solver3 import RiemannSolver3
 from fv3core.stencils.riem_solver_c import RiemannSolverC
-from pace.dsl.stencil import StencilFactory
+from pace.dsl.stencil import GridIndexing, StencilFactory
 from pace.dsl.typing import FloatField, FloatFieldIJ, FloatFieldK
-from pace.stencils.testing.grid import (
-    DampingCoefficients,
-    GridData,
-    GridIndexing,
-    axis_offsets,
-    quantity_wrap,
-)
 from pace.util import X_DIM, Y_DIM, Z_DIM, Z_INTERFACE_DIM
+from pace.util.grid import DampingCoefficients, GridData, quantity_wrap
```

Comment:
It appears this is actually the only place we use `quantity_wrap`. The function itself is a pretty private-looking one, that always sets units to "unknown" - does it make sense to put that function directly in this file with a `_` starting its name?